lib/ja_serializer/builder/resource_identifier.ex

```diff
@@ -48,4 +49,21 @@ defmodule JaSerializer.Builder.ResourceIdentifier do
     end
   end
 
+  defp get_meta(
+    destination_data,
+    %{data: source_data, serializer: serializer, conn: conn},
+    %{identifier_meta: identifier_metadata_name})
+  when is_atom(identifier_metadata_name) do
+    apply(serializer, identifier_metadata_name, [source_data, destination_data, conn])
+  end
+
+  defp get_meta(
+    destination_data,
+    %{data: source_data, conn: conn},
+    %{identifer_meta: identifier_metadata_func})
+  when is_function(identifier_metadata_func) do
+   identifier_metadata_func.(source_data, destination_data, conn)
```

Comment:
Hice, does this work? I have had some trouble getting functions working when passed to the DSL.